src/custom-blocks/iframe/index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * iFrame block
+ * Frontend PHP code
+ *
+ * Uses WordPress' dynamic block method
+ * https://developer.wordpress.org/block-editor/tutorials/block-tutorial/creating-dynamic-blocks/
+ *
+ * @package mojblocks
+ *
+ */
+
+function render_callback_iFrame_block($attributes, $content)
+{
+    // Parse attributes found in index.js
+    $attribute_iFrame_url = $attributes['iFrameURL'] ?? '';
+    $attribute_iFrame_width = $attributes['iFrameWidth'] ?? '';
+    $attribute_iFrame_height = $attributes['iFrameHeight'] ?? '';
+    $attribute_iFrame_border = $attributes['iFrameBorder'] ?? false;
+    $attribute_iFrame_centre = $attributes['iFrameCentre'] ?? false;
+    $attribute_iFrame_className = $attributes['iFrameClassName'] ?? '';
+
+    if ($attribute_iFrame_url == "" || substr($attribute_iFrame_url,0,8) != "https://") {
+        return "";
+    }
+    
+    $attribute_iFrame_url = esc_url($attribute_iFrame_url);
+    $attribute_iFrame_width = esc_html($attribute_iFrame_width);
+    $attribute_iFrame_height = esc_html($attribute_iFrame_height);
```

Comment:
These two, width and height, check you can use esc_html on numbers

Reply:
Yeah, no point having these is there?  I've removed.